product-service/src/db/task4script.sql

```diff
@@ -0,0 +1,55 @@
+create extension if not exists "uuid-ossp";
+
+create table product (
+	id uuid not null default uuid_generate_v4() primary key,
+	title text not null,
+	description text,
+	price integer
+);
+
+create table stock (
+	product_id uuid not null primary key,
+	count integer,
+	foreign key (product_id) references product(id)
+);
+
+
+INSERT INTO public.product
+(id, title, description, price)
+VALUES('7567ec4b-b10c-48c5-9345-fc73c48a80aa', 'product title1', 'This product1', 1);
```

Comment:
@AmberCode Why do you insert `id` explicitly in `product` table if you already have `uuid_generate_v4()` function? You can omit it.